crates/edr_evm/src/blockchain/forked.rs

```diff
@@ -576,6 +581,13 @@ where
             })?)
         }
     }
+
+    fn base_fee_params(&self) -> ConstantBaseFeeParams {
+        *self
+            .base_fee_params
+            .at_hardfork(self.hardfork())
+            .expect("Chain spec must have base fee params for post-London hardforks")
```

Comment:
It seems safer to have a default to revert to if the hardfork is not present. Since we won't have control over the defined base fee parameters, we would now be panicking.